cmake/resources/generate_resources.cmake

```diff
@@ -78,7 +77,16 @@ endforeach ()
 list(JOIN RES_HEADER_CONTENT "\n" RES_HEADER_CONTENT)
 configure_file(${CMAKE_CURRENT_LIST_DIR}/ResourcesAutogen.hpp.in ${CMAKE_BINARY_DIR}/autogen/ResourcesAutogen.hpp @ONLY)
 
-set(RES_AUTOGEN_FILES
+# clang-cl doesn't support resource files
```

Comment:
This is fake news. This was first added in 2e2846c83d2cb192e1563c43209ef6be6ef4311f, but the current `clang-cl` supports resource files (`-DCMAKE_CXX_COMPILER=clang-cl -DCMAKE_C_COMPILER=clang-cl -DCMAKE_RC_COMPILER=llvm-rc`, `llvm-rc` is optional). I believe the last two or three LLVM versions can all build Chatterino on Windows. Currently, `sccache` doesn't work, as it passes `/wd4668`, and the crashpad build doesn't work (might want to make an issue, but it's not really important).

Reply:
KKonaW my kind of news
clang exclusion removed ed37f053c364ad21b42ad59fa50e9907ff304c3d